src/TeamSyncer.ts

```diff
@@ -207,13 +207,17 @@ export class TeamSyncer {
         }
     }
 
-    public async syncUser(teamId: string, domain: string, item: ISlackUser) {
+    public async syncUser(teamId: string, domain: string, item: ISlackUser, initial = false) {
         log.info(`Syncing user ${teamId} ${item.id}`);
         const slackGhost = await this.main.ghostStore.get(item.id, domain, teamId);
```

Comment:
This function creates a ghost, if none does exist.
Do we need to check `item.deleted === true && initial` before calling `ghostStore.get()` to avoid creating a ghost that we don't want?